invalid yaml

hello
we test all the files in our puppet files repository with out CI/CD system

yamllint shows that the yaml files are missing the "---" headings, and a problem with a blank line



/tmp/build/32643510/devel-branch/environments/production/modules/gitlab/data/family/RedHat.yaml
  2:1       warning  missing document start "---"  (document-start)
  22:1      error    too many blank lines (1 > 0)  (empty-lines)

yamllint found error in /tmp/build/32643510/devel-branch/environments/production/modules/gitlab/data/family/RedHat.yaml when using config /tmp/build/32643510/ci-scripts/ci/yamllint-config.yaml

===============================================================================
Checking /tmp/build/32643510/devel-branch/environments/production/modules/gitlab/data/family/RedHat/6.yaml
/tmp/build/32643510/devel-branch/environments/production/modules/gitlab/data/family/RedHat/6.yaml
  1:1       warning  missing document start "---"  (document-start)


===============================================================================
Checking /tmp/build/32643510/devel-branch/environments/production/modules/gitlab/data/os/Ubuntu.yaml
/tmp/build/32643510/devel-branch/environments/production/modules/gitlab/data/os/Ubuntu.yaml
  1:1       warning  missing document start "---"  (document-start)

--- a/data/family/Debian.yaml
+++ b/data/family/Debian.yaml
@@ -1,3 +1,5 @@
+---
+
 gitlab::repository_configuration:
   'apt::source':
     "gitlab_official_ce":

--- a/data/family/RedHat.yaml
+++ b/data/family/RedHat.yaml
@@ -1,3 +1,5 @@
+---
+
 # From: https://packages.gitlab.com/install/repositories/gitlab/gitlab-ce/config_file.repo?os=centos&dist=7&source=script
 gitlab::repository_configuration:
   yumrepo:
@@ -19,4 +21,3 @@ gitlab::repository_configuration:
       gpgcheck: 1
       repo_gpgcheck: 1
       sslverify: 1
-

--- a/data/family/RedHat/6.yaml
+++ b/data/family/RedHat/6.yaml
@@ -1,1 +1,3 @@
+---
+
 gitlab::service_enable: false

--- a/data/os/Ubuntu.yaml
+++ b/data/os/Ubuntu.yaml
@@ -1,3 +1,5 @@
+---
+
 gitlab::repository_configuration:
   'apt::source':
     "gitlab_official_ce":

